### PR TITLE
[Exporter.InfluxDB] Update CHANGELOG for 1.0.0-alpha.1 release

### PR DESCRIPTION
**Pull request by Havret**
## Changes

- Update CHANGELOG for 1.0.0-alpha.1 release

### CODE REVIEW

File: src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,5 @@
 # Changelog
+
+## 1.0.0-alpha.1
+
+Released 2023-May-18
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md:5]: Add some text under this. Refer to [this](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.Runtime/README.md) for example.

File: src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md [hunk 2]
```diff
@@ -1 +1,5 @@
 # Changelog
+
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md:2]: Add a header for `Unreleased` to keep track of newer unreleased changes.

**Review by utpilla — APPROVED** [review]

**Comment by utpilla** [comment]
The package is published: https://www.nuget.org/packages/OpenTelemetry.Exporter.InfluxDB/1.0.0-alpha.1

File: src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md (hunk 2 above)
Comment (Havret) [reply src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md:2]: Fixed with #1197.

File: src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md (hunk 1 above)
Comment (Havret) [reply src/OpenTelemetry.Exporter.InfluxDB/CHANGELOG.md:5]: Fixed with #1197.

**Pull request merged**